docs/reference/ccr/apis/auto-follow/delete-auto-follow-pattern.asciidoc

```diff
@@ -5,7 +5,7 @@
 <titleabbrev>Delete auto-follow pattern</titleabbrev>
 ++++
 
-Delete auto-follow patterns.
+Delete {ccr} <<ccr-auto-follow,auto-follow patterns>>.
```

Comment:
updated this to link to the auto-follow pattern docs instead of the ccr docs. if someone needs a ccr link they can get it from the ccr api overview page.

commenting this for visibility for future work - I think we should link to the closest conceptual doc match for the actual item the API impacts/mutates.

Reply:
Agreed! Thanks I missed there was a CCR auto-follow page 🙂